libutils/file_lib.h

```diff
@@ -102,9 +102,24 @@ NewLineMode FileNewLineMode(const char *file);
 # define FILE_SEPARATOR_STR "/"
 #endif
 
-StringSet* GlobFileList(const char *pattern);
-
 bool IsAbsoluteFileName(const char *f);
+
+/**
+ * @brief Check whether or not path is a Windows network path.
+ * @param path Path to check.
+ * @return True if path is network path, otherwise false.
+ * @note This function cannot return true on non-Windows platforms.
+ */
+bool IsNetworkPath(const char *path);
```

Comment:
Since this is something Windows-specific I'd put that in the name:

```suggestion
bool IsWindowsNetworkPath(const char *path);
```